code/datums/weakrefs.dm

```diff
@@ -16,8 +16,10 @@
 /datum/weakref/New(datum/thing)
 	reference = REF(thing)
 
-/datum/weakref/Destroy()
-	return QDEL_HINT_LETMELIVE	//Let BYOND autoGC thiswhen nothing is using it anymore.
+/datum/weakref/Destroy(force)
+	if(!force)
+		return QDEL_HINT_LETMELIVE	//Let BYOND autoGC thiswhen nothing is using it anymore.
+	return ..()
```

Comment:
Should probably clean up the `weakref` var which points to it or else it will harddel